proposals/gc/Overview.md

```diff
@@ -579,11 +580,6 @@ TODO: The ability to import types makes the type and import sections interdepend
 TODO
 
 
-## Possible Extension: Closures
```

Comment:
Please don't remove prematurely. There are a number reasons why we still might want closures as a primitive mechanism, see e.g. #7, or language interop, or more efficient support for currying, etc.